components/TaskCard.vue

```diff
@@ -30,6 +31,18 @@ export default {
       default: null,
     },
   },
+  data: () => ({
+    task: '',
+    loading: false,
+  }),
+  mounted() {
+    this.task = _.clone(this.cardData);
```

Comment:
Why do we need to clone this here?

Reply:
It throughs an error saying i am editing the vuex state outside the store while i try to edit the task description

Reply:
Are you by any chance trying to mutate store state inside a component?